src/content/blog/images.mdx

```diff
@@ -0,0 +1,153 @@
+- For accessibility, the required `alt` prop helps improve your site for screen-readers (and other tooling).
+- For even faster page-load performance, the `loading` and `decoding` attributes are added as defaults.
+- For the user experience, The `width` and `height` attributes are always included to prevent layout shift (more on this later).
+
+Flexibility was another goal. We added the [`getImage()`](https://docs.astro.build/en/guides/assets/#getimage-astroassets) function to optimize an image in your project, on-demand, without using the `<Image />` component. Use this function to build your own custom Image component for any use-case you you have in mind.
+
+```astro
+---
+// Example: Instead of creating an `img` tag,
+// apply an optimized image to a div using CSS.
+import { getImage } from "astro:assets"
+import myImage from "../assets/penguin.png"
+const optimizedBackgroundImage = await getImage({ src: myImage })
+---
+
+<div style={`background-image: url('${optimizedBackgroundImage.src}')`}></div>
+```
+
+```html
+<!-- Result: -->
+<div style="background-image: url('/_astro/penguin.XXXXXX.webp')"></div>
+```
+
+## Automatic layout shift prevention
+
+[Cumulative Layout Shift (CLS)](https://web.dev/cls/) is an essential metric to optimize for website performance. It is tracked as one of Google's 3 [Core Web Vitals](https://web.dev/vitals/) and is weighted heavily in the [Lighthouse](https://developer.chrome.com/docs/lighthouse/overview/) performance score.
+
+**Astro's new `<Image />` component automatically protects your site from layout shift.** It works by inferring an explicit height and width for every image and including them in the generated `<img>` tag. With explicit dimensions, the image can't shift the page layout as it loads onto the page.
+
+The output HTML is always a simple `<img>` element, so you can also apply additional CSS styling for more advanced responsive layouts like `fill` and `cover` without risking layout shift.
+
+```astro
+<Image
+	src={myImage}
+	alt="A very cool penguin!"
+	style="height: 100%; width: 100%; object-fit: cover;"
+/>
+```
+
+## Automatic Markdown & MDX support
+
+One of the biggest improvements made over the original `@astrojs/image` integration is complete support for Markdown, Markdoc, and MDX. You can now reference images using a relative image path in your Markdown content or frontmatter, and Astro will automatically optimize them for you.
+
+```md
+![A starry night sky](../../assets/stars.png)
+```
+
+```html
+<!-- Result: -->
+<img
+	src="/_astro/stars.XXXXXX.webp"
+	width="300"
+	height="300"
+	loading="lazy"
+	decoding="async"
+	alt="A starry night sky"
+/>
+```
+
+That's it! Your `stars.png` image will be optimized automatically in the final HTML, as if you had used the `<Image>` component directly. You can also use the `<Image />` component directly in MDX if you prefer, but in most cases the default settings will be enough to produce an optimized file size with a great image quality.
```

Comment:
```suggestion
That's it! Your `stars.png` image will be optimized automatically in the final HTML. You can also use the `<Image />` component directly in MDX if you prefer for finer control over the image attributes, but in most cases the default settings will be enough to produce an optimized file size with a great image quality.
```
Tiny bit repetitive with "`<Image>` component directly", and I think it's more helpful to explain why you might Ithink you) want the image component.